index.bs

```diff
@@ -5920,16 +5920,16 @@ implement, as they determine the characteristics of the different
 filter types. They are inspired by formulas found in the
 <a href="https://www.w3.org/2011/audio/audio-eq-cookbook.html">Audio EQ Cookbook</a>.
 
-The transfer function for the filters implemented by the
-{{BiquadFilterNode}} is:
+The {{BiquadFilterNode}} processes audio with a time-domain equation of:
 
 <pre nohighlight>
 $$
-H(z) = \frac{\frac{b_0}{a_0} + \frac{b_1}{a_0}z^{-1} + \frac{b_2}{a_0}z^{-2}}
-						{1+\frac{a_1}{a_0}z^{-1}+\frac{a_2}{a_0}z^{-2}}
+y(n) + \frac{a_1}{a_0} y(n-1) + \frac{a_2}{a_0} y(n-2) =
```

Comment:
This is inconsistent with the IIRFilter. I think it's best to make them consistent.  (Change this or change the IIRFIlter.)

Reply:
Wait, how? I moved the y(n) to the LHS, I'm not sure how much more consistent you want it. Do you want it to use summation notation?

Reply:
Sorry for being unclear.  The IIRFilter formula is, basically

```
a0*y(n) + a1*y(n-1) + ... = b0*x(n) + b1*x(n-1) + ...
```
You have
```
y(n) + a1/a0*y(n-1) + ... = b0/a0 * x(n-1) + b1/a0*x(n-1) + ...
```
I'm fine with either form. I just think the Biquad and IIR should use the same form for the time-domain equations.

Reply:
fixed